README.md

```diff
@@ -1,5 +1,13 @@
-This repository contains a list of tests in the Salt Test Suite that are **not** executed. Skipping
-tests is bad and should only be used as a last resort.
+This repository contains a list of tests in the Salt Test Suite that are **not** executed. Remember: **Skipping tests is bad and should only be used as a last resort.**
 
-The list is formatted using the TOML markup language and skipped tests are grouped by file/test level and then by "type". Missing
-groups do  not have t be listed explicitely.
\ No newline at end of file
+The list is formatted using the TOML markup language and skipped tests are grouped by file/test level and then by "type". Missing groups do not have to be listed explicitely. The list is used by `salt-test`, a test launcher that skips the collection of files in the `[ignore]` sections and deselects all tests in the `[skip]` section.
+
+### `ignore` vs `skip`
+
+Rule of thumb: Use `skip` if you can, `ignore` if you must.
+Reason: `ignore` skips over a whole Python module (i.e. a file). It's needed when the import of the module causes errors, e.g. `ImportError`. We don't have all of Salt's Python dependencies packaged and need to exclude that's in this manner that don't guard their imports.
```

Comment:
Is the wording in the last sentence correct here?

Reply:
No, it's not :facepalm: Good catch!